### PR TITLE
hexdump: add pt_BR translation

### PR DESCRIPTION
**Pull request by paulosell**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Comment by paulosell** [comment]
@waldyrious

File: pages.pt_BR/common/hexdump.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# hexdump
+
+> Impressão de dados no formato ASCII, decimal, hexadecimal ou octal.
+> Mais informações: <https://manned.org/hexdump>.
+
+- Imprime a representação hexadecimal de um arquivo, substituindo linhas duplicadas por '*':
+
+`hexdump {{nome_do_arquivo}}`
```
Comment (kbdharun) [inline pages.pt_BR/common/hexdump.md:8]:
```suggestion
`hexdump {{caminho/para/arquivo}}`
```

File: pages.pt_BR/common/hexdump.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+- Imprime a representação hexadecimal e ASCII de um arquivo, em duas colunas:
+
+`hexdump -C {{nome_do_arquivo}}`
```
Comment (kbdharun) [inline pages.pt_BR/common/hexdump.md:12]:
```suggestion
`hexdump -C {{caminho/para/arquivo}}`
```

File: pages.pt_BR/common/hexdump.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+- Imprime a representação hexadecimal de um arquivo, porém apresentando apenas seus n primeiros bytes:
+
+`hexdump -C -n{{numero_de_bytes}} {{nome_do_arquivo}}`
```
Comment (kbdharun) [inline pages.pt_BR/common/hexdump.md:16]:
```suggestion
`hexdump -C -n{{numero_de_bytes}} {{caminho/para/arquivo}}`
```

File: pages.pt_BR/common/hexdump.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+- Não substitui as linhas duplicadas por '*':
+
+`hexdump --no-squeezing {{nome_do_arquivo}}`
```
Comment (kbdharun) [inline pages.pt_BR/common/hexdump.md:20]:
```suggestion
`hexdump --no-squeezing {{caminho/para/arquivo}}`
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

Ref. https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/common-arguments.md

File: pages.pt_BR/common/hexdump.md (hunk 1 above)
Comment (paulosell) [reply pages.pt_BR/common/hexdump.md]: Fixed

File: pages.pt_BR/common/hexdump.md (hunk 2 above)
Comment (paulosell) [reply pages.pt_BR/common/hexdump.md]: Fixed

File: pages.pt_BR/common/hexdump.md (hunk 3 above)
Comment (paulosell) [reply pages.pt_BR/common/hexdump.md]: Fixed

File: pages.pt_BR/common/hexdump.md (hunk 4 above)
Comment (paulosell) [reply pages.pt_BR/common/hexdump.md]: Fixed

**Review by kbdharun — APPROVED** [review]

File: pages.pt_BR/common/hexdump.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
+# hexdump
+
+> Impressão de dados no formato ASCII, decimal, hexadecimal ou octal.
```
Comment (waldyrious) [inline pages.pt_BR/common/hexdump.md:3]:
```suggestion
> Imprime dados no formato ASCII, decimal, hexadecimal ou octal.
```

File: pages.pt_BR/common/hexdump.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+> Mais informações: <https://manned.org/hexdump>.
+
+- Imprime a representação hexadecimal de um arquivo, substituindo linhas duplicadas por '*':
```
Comment (waldyrious) [inline pages.pt_BR/common/hexdump.md:6]:
```suggestion
- Imprimir a representação hexadecimal de um arquivo, substituindo linhas duplicadas por '*':
```

File: pages.pt_BR/common/hexdump.md [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+
+`hexdump {{caminho/para/arquivo}}`
+
+- Imprime a representação hexadecimal e ASCII de um arquivo, em duas colunas:
```
Comment (waldyrious) [inline pages.pt_BR/common/hexdump.md:10]:
```suggestion
- Imprimir a representação hexadecimal e ASCII de um arquivo, em duas colunas:
```

File: pages.pt_BR/common/hexdump.md [hunk 8]
```diff
@@ -0,0 +1,20 @@
...
+
+`hexdump -C {{caminho/para/arquivo}}`
+
+- Imprime a representação hexadecimal de um arquivo, porém apresentando apenas seus n primeiros bytes:
```
Comment (waldyrious) [inline pages.pt_BR/common/hexdump.md:14]:
```suggestion
- Imprimir a representação hexadecimal de um arquivo, porém apresentando apenas seus n primeiros bytes:
```

**Comment by waldyrious** [comment]
Are you learning some Portuguese, @kbdharun? 😉

Hey @paulosell, my apologies for taking a long time to get to this. I was going through a busy period and didn't have the mental bandwidth to review PRs with the due attention. I will now take care of this. Hope it wasn't too burdensome a wait!

File: pages.pt_BR/common/hexdump.md [hunk 9]
```diff
@@ -0,0 +1,20 @@
...
+
+`hexdump -C -n{{numero_de_bytes}} {{caminho/para/arquivo}}`
+
+- Não substitui as linhas duplicadas por '*':
```
Comment (waldyrious) [inline pages.pt_BR/common/hexdump.md:18]:
```suggestion
- Imprimir a representação hexadecimal completa de um arquivo (sem omitir linhas duplicadas):
```

**Comment by kbdharun** [comment]
> Are you learning some Portuguese, @kbdharun? 😉

Maybe 😄

Just kidding referred to the common arguments page.

**Review by waldyrious — CHANGES_REQUESTED** [review]
Great work overall! I'm leaving just a few minor change suggestions.

Note: I realize the standard for the verbal tense to use hasn't been set in stone yet, so apologies if the existing pages gave you the wrong impression about what form to use for the descriptions.

**Comment by paulosell** [comment]
@waldyrious all done! 😄

**Review by waldyrious — APPROVED** [review]
Great job, @paulosell! Thanks for the contribution, and sorry again for the delay! LGTM :+1:

**Pull request merged**